### PR TITLE
[FAN-230] Change NFT update model

### PR DESCRIPTION
**Pull request by sidorovkirill**
https://fangible.atlassian.net/browse/FAN-230

- [x] I've performed a code self-review
- [x] I've tested feature or bugfix on my own
- [x] If I make changes to another person's module, I've asked how to use it or request a review
- [x] I've checked mobile & tablet screens (if have changes in UI)
- [x] Local build successfully completed
- [x] Branch name and pull request title are according to [the requirements](https://fangible.atlassian.net/wiki/spaces/FAN/pages/62226433/Branches+naming)

### CODE REVIEW

File: src/modules/brand/actions/queryBrandNfts.ts [hunk 1]
```diff
@@ -140,6 +140,7 @@ export const queryBrandNfts = createAction<
             })
             .filter(item => item && item.itemname) as INFTItem[];
 
+          // @ts-ignore
```
Comment (Antroll) [inline src/modules/brand/actions/queryBrandNfts.ts]: Is it necessary?

File: src/modules/brand/actions/queryBrandPools.ts [hunk 2]
```diff
@@ -126,6 +126,7 @@ export const queryBrandPools = createSmartAction<RequestAction>(
               PromiseList.push(queryEaList);
             }
 
+            // @ts-ignore
```
Comment (Antroll) [inline src/modules/brand/actions/queryBrandPools.ts]: Is it necessary?

**Review by Antroll — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -67,7 +67,8 @@
     "@types/react-truncate": "^2.3.3",
     "babel-loader": "8.1.0",
     "husky": "^7.0.0",
-    "lint-staged": "^10.5.4"
+    "lint-staged": "^10.5.4",
+    "madge": "^5.0.1"
```
Comment (BigBadAlien) [inline package.json:71]: 😮

File: package.json [hunk 4]
```diff
@@ -83,7 +84,8 @@
     "eject": "react-scripts eject",
     "format": "prettier --write \"**/*.{js,jsx,ts,tsx}\"",
     "prepare": "husky install",
-    "lint-staged": "lint-staged"
+    "lint-staged": "lint-staged",
+    "check-circular-dependency": "madge --circular src"
```
Comment (BigBadAlien) [inline package.json:88]: 😮

**Pull request merged**